README.md

```diff
@@ -18,10 +18,198 @@ Or install it yourself as:
 
     $ gem install litmus_paper
 
+## Overview
+
+Litmus Paper reports health for each service on a node on a 0-100 scale. Health is computed by aggregating the values returned from running various subchecks.
+
+There are two classes of subchecks: Dependencies and Metrics. Dependencies report either 0 or 100 health, and aggregate such that if any dependency is not 100 the whole service is down. Metrics report health on a scale from 0-100 and aggregate as averages based on their weight.
+
+You can also force a service to report a health value on a host.
+Forcing a service up makes it report a health of 100, regardless of the measured health.
+Forcing a service down makes it report a health of 0.
+Forcing a service's health to a value between 0 and 100 causes it to report that value.
+
+Force downs take precedence, followed by force ups, and finally force healths. If you specify both a force up and a force down, the service will report 0 health. If you specify a force up and a force health, the force health will be ignored and the service will report 100 health.
+
+Using litmus-agent-check, Litmus can also output health information in haproxy agent check format.
+
 ## Usage
 
-Use the sample config to run it under unicorn. Or execute litmus-agent-check to
-use it as an agent-check for HAProxy.
+### Running Litmus Paper
+
+Start the process under unicorn with `/usr/bin/litmus --unicorn-config <path_to_unicorn_conf>`. In the unicorn config file, set the number of worker processes, the pid, and the working directory. See the [unicorn documentation](https://bogomips.org/unicorn/Unicorn/Configurator.html) for the config format. There are also more options, see `/usr/bin/litmus -h` for a full list.
```

Comment:
Maybe we can include a sample unicorn config?  I know we have one in `spec/support/test.unicorn.config`, but it might be nice to make it a little more explicit.

Reply:
I added a simple unicorn config in https://github.com/braintree/litmus_paper/pull/34